components/configs/rwn-optimizations/base/3-node/worker-fix-ipi-rwn.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 50-worker-fix-ipi-rwn
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:,
+            verification: {}
+          filesystem: root
+          mode: 420
+          path: /etc/kubernetes/manifests/keepalived.yaml
+        - contents:
+            source: data:,
+            verification: {}
+          filesystem: root
+          mode: 420
+          path: /etc/kubernetes/manifests/mdns-publisher.yaml
+        - contents:
+            source: data:,
+            verification: {}
+          filesystem: root
+          mode: 420
+          path: /etc/kubernetes/manifests/coredns.yaml
```

Comment:
This too I think is no longer necessary post 4.8.  I believe originally coredns generated mDNS traffic in the cluster, which wasn't something we wanted.

Reply:
> Since I see `machineconfig` here, do we care about the ordering?

I believe not in this specific case. We need to try.